### ISSUE TITLE
Demo Site Examples

### ISSUE THREAD

**Issue by PaulKish** [jdewit/bootstrap-timepicker#305]
Demo site examples seem to broken. Using Firefox 41.0.2

**Comment by mrhota** [comment]
You're right. I don't have the tools set up to accurately repro the demo site locally. So it's sort of janky for now.

**Comment by IvoPereira** [comment]
+1 here. It would be nice to see this fixed, as not seeing any examples of implementation would possibly get me choosing another snippet unfortunately.

**Comment by rloveless** [comment]
+1, it's hard to trust code when there's no working examples.
